### ISSUE TITLE
[Fx115] ac_popup_megabar_title_and_url_two_lines.css does not work

### ISSUE THREAD

**Issue by f-c-s** [Aris-t2/CustomCSSforFx#703]
Hello,

In versions 4.5.3 and 4.5.9 the

@import "./css/locationbar/ac_popup_megabar_title_and_url_two_lines.css"

does not work in FF 115.17 esr.

The version coming from 4.1.8 works fine.

Is there a way to increase the items shown in drop-down list after clicking on mega-bar ?
By default only 6-8 items are visible.

Regards
ML

**Comment by Aris-t2** [comment]
You can increase the amount of results using `about:config > browser.urlbar.maxRichResults > 20`.

If the file from an older build works, use it for now.
Most up to date version of `ac_popup_megabar_title_and_url_two_lines.css seems` to word on Fs 128ESR+

**Comment by CT-ABT** [comment]
The last version of this CSS project that supports Firefox 115 is actually ver 4.4.2. I have noticed this problem a long while ago ;) you can copy the code below and replace all the contents of the current version’s `ac_popup_megabar_title_and_url_two_lines.css` to make it work again:
```css
/* Firefox userChrome.css tweaks ********************************************************/
/* Github: https://github.com/aris-t2/customcssforfx ************************************/
/****************************************************************************************/


/* based on [wrap]-code from default browser.css */

#urlbar .urlbarView-results > .urlbarView-row > .urlbarView-row-inner {
  flex-wrap: wrap !important;
}
#urlbar .urlbarView-results > .urlbarView-row > .urlbarView-row-inner > .urlbarView-no-wrap {
  max-width: 100% !important;
  flex-basis: 100% !important;
}
#urlbar .urlbarView-results > .urlbarView-row[has-url] > .urlbarView-row-inner > .urlbarView-url {
  margin-top: 2px !important;
  padding-inline-start: calc(6px + 1px + 16px) !important;
}

#urlbar[actionoverride] .urlbarView-results > .urlbarView-row[has-url] > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-title-separator,
#urlbar .urlbarView-results > .urlbarView-row[has-url]:not([type$=tab]) > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-title-separator {
  display: none !important;
}

/* support for heuristic items: search, remote tab, switch to tab */
#urlbar .urlbarView-results > .urlbarView-row:is([type=search],[type=switchtab]) > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-title-separator,
#urlbar .urlbarView-results > .urlbarView-row:not([has-url]) > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-title-separator {
  display: none !important;
}

#urlbar .urlbarView-results > .urlbarView-row:is([type=search],[type=switchtab]) > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-action,
#urlbar .urlbarView-results > .urlbarView-row:not([has-url]) > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-action,
#urlbar .urlbarView-results > .urlbarView-row:is([type=remotetab]):not([selected]):not(:hover) > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-action {
  display: block !important;
  position: absolute !important;
}

#urlbar .urlbarView-results > .urlbarView-row:is([type=search],[type=switchtab]) > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-action,
#urlbar .urlbarView-results > .urlbarView-row:not([has-url]) > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-action {
  padding-inline-start: 24px !important;
}

#urlbar .urlbarView-results > .urlbarView-row:is([type=remotetab]):not([selected]):not(:hover) > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-action {
  margin-bottom: -47px !important;
  padding-inline-start: 16px !important;
}

#urlbar .urlbarView-results > .urlbarView-row[type=switchtab] > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-action {
  margin-bottom: -47px !important;
  appearance: none !important;
  background: unset !important;
  padding-inline-start: 15px !important;
}

#urlbar .urlbarView-results > .urlbarView-row[type=remotetab] > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-action {
  appearance: none !important;
  background: unset !important;
}

#urlbar .urlbarView-row[dynamicType=onboardTabToSearch] > .urlbarView-row-inner {
  min-height: unset !important;
}

.urlbarView-row[dynamicType=onboardTabToSearch] > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-favicon {
  min-width: unset !important;
  height: unset !important;
}

.urlbarView-dynamic-onboardTabToSearch-description {
  display: none !important;
}

#urlbar .urlbarView-results > .urlbarView-row[has-url]:is([type=remotetab],[sponsored],[type=switchtab],[type=search]) > .urlbarView-row-inner > .urlbarView-url {
  display: none !important;
}

.urlbarView-row[rich-suggestion] > .urlbarView-row-inner > .urlbarView-favicon {
  width: 16px !important;
  height: 16px !important;
  flex-basis: 16px !important;
}

.urlbarView-row:not([rich-suggestion]),
.urlbarView-row[rich-suggestion][icon-size="16"] {
  --urlbarView-icon-margin-start: 0px !important;
  --urlbarView-icon-margin-end: 7px !important;
}

.urlbarView-row[label]::before {
  display: none !important;
}

.urlbarView-row[label] {
  margin-block-start: unset !important;
}

#urlbar .urlbarView-button-menu {
  z-index: 1 !important;
}

#urlbar .urlbarView-results > .urlbarView-row[rich-suggestion],
#urlbar .urlbarView-results > .urlbarView-row[role="presentation"]:is([type=search]) > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-action,
#urlbar .urlbarView-results > .urlbarView-row[role="presentation"]:not([has-url]) > .urlbarView-row-inner > .urlbarView-no-wrap > .urlbarView-action,
.urlbarView-row[role="presentation"][type="search"] .urlbarView-action {
  display: none !important;
}
```

**Comment by Aris-t2** [comment]
I guess we have to tag Fx 115 as lagacy, since the current release is mainly for 128+.

**Comment by Aris-t2** [comment]
Your file is part for this projects legacy files: https://github.com/Aris-t2/CustomCSSforFx/blob/master/legacy/fx109-127/css/locationbar/ac_popup_megabar_title_and_url_two_lines_fx115.css

**Comment by f-c-s** [comment]
Hello,

Thank you.

I reopen this topic due to I found the solution for the number of items displayed after clicking urlBar.

I had set the variable 

about:config > browser.urlbar.maxRichResults =20

before I asked about that. So it is not the solution, but it is a part of it. 

It plays together with the number of rows, which are set on browser standard settings:

Settings > Starting > Start page > ShortCuts number of rows = 1....4,

By default it is set to 1 row, which contains 8 items. So clicking on urlBar these 8 items are displayed
If the number of rows is set to 4 it means 32 items (4x8) which may be displayed.
Having set 2 lines for one item on drop-down urlBar, it finally means about 23 items, which are limited by maxRichResults i,e. into 20 items like above.

https://support.mozilla.org/en-US/kb/customize-your-new-tab-page
https://support.mozilla.org/en-US/kb/customize-items-on-firefox-new-tab-page#w_show-more-or-fewer-shortcuts

The other way to display more items (although the ShorCut is set into 1 row by default), is pressing the "Space" on urlBar.
In this case the number of items are always limited by maxRichResults by default set into 10.

Digging internet I found that the char ^ works similar to space, but space determines the most visited pages but ^ most searched pages, but at this moment I can't confirm this.  I'm interested in most visited pages.

Regards
ML
